src/applications/vaos/new-appointment/components/ReviewPage/VAAppointmentSection.jsx

```diff
@@ -23,12 +35,10 @@ export default function VAAppointmentSection({ data, facility }) {
             {TYPE_OF_VISIT.find(visit => visit.id === data.visitType)?.name}
           </div>
           <div>
-            <Link
-              to={newAppointmentFlow.visitType.url}
+            <va-link
+              onClick={handleClick(history, dispatch)}
```

Comment:
This is missing the `Edit` link
```suggestion
            <va-link
              onClick={handleClick(history, dispatch)}
              text="Edit"
```